Extension/package.json

```diff
@@ -2361,6 +2361,37 @@
           "description": "%c_cpp.configuration.doxygen.generatedStyle.description%",
           "scope": "resource"
         },
+        "C_Cpp.doxygen.selectDoxygenTags": {
+          "type": "array",
+          "items": {
+            "type": "string",
+            "enum": [
```

Comment:
The tags that are currently hard-coded in the language server should be available in this enum.  It looks like some of them are missing.  The user should have the ability to remove them.